src/webui/www/package.json

```diff
@@ -6,13 +6,15 @@
     "url": "https://github.com/qbittorrent/qBittorrent.git"
   },
   "scripts": {
+    "extract_translation": "i18next -c public/i18next-parser.config.mjs public/index.html public/scripts/login.js",
```

Comment:
So, do we need to add each file here separately later?

Reply:
It might support wildcards though.
Also it would be better to append another command to this script: `... && i18next -c private/i18next-parser.config.mjs private/*.html ...`

Reply:
Yes, it supports globbing patterns as evidence in their readme examples: https://github.com/i18next/i18next-parser